src/plugins/chart_expressions/expression_heatmap/public/components/heatmap_component.test.tsx

```diff
@@ -30,15 +30,6 @@ import HeatmapComponent from './heatmap_component';
 import { LegendSize } from '@kbn/visualizations-plugin/common';
 import { FieldFormat } from '@kbn/field-formats-plugin/common';
 
-jest.mock('@elastic/charts', () => {
-  const original = jest.requireActual('@elastic/charts');
-
-  return {
-    ...original,
-    getSpecId: jest.fn(() => {}),
-  };
-});
```

Comment:
This is now superfluous. Chart `ids` used to be cast as unique strings via the `getSpecId` helper (see [example](https://github.com/markov00/elastic-charts/blame/8afa62afc4206d8ce3876aa5ccd9c69a8c20518a/src/utils/ids.ts)) but this was changed in  https://github.com/elastic/elastic-charts/pull/281 and fully removed in https://github.com/elastic/elastic-charts/pull/554.